templates/cli/index.js.twig

```diff
@@ -28,12 +28,24 @@ program
   .version(version, "-v, --version")
   .option("--verbose", "Show complete error log")
   .option("--json", "Output in JSON format")
+  .option("-f,--force", "Flag to confirm all warnings")
+  .option("-a,--all", "Flag to push all resources")
+  .option("--ids [id...]", "Flag to pass list of ids for a giving action")
```

Comment:
Hmm not sure if it should be `--id` or `--ids`. Christy to decide

Reply:
Prefer:
appwrite push function --id createPayment --id deletePayment

If not supported, then
appwrite push function --ids createPayment deletePayment